applications/blockchainia.md

```diff
@@ -0,0 +1,139 @@
+
+## Team 
+
+### Team members
+
+- Ed Anderson (eca20@pitt.edu)
+- Will Chastka (wchastka@tepper.cmu.edu)
+- Matt Dennis (oopmatt@gmail.com)
+
+### Contact
+
+- **Contact Name:** Ed Anderson
+- **Contact Email:** eca20@pitt.edu
+- **Website:** https://github.com/eca20
+
+### Legal Structure
+
+- **Registered Address:** 502 W 7th St, Suite 100, Erie, PA 16502
+- **Registered Legal Entity:** Blockchainia LLC
+
+### Team's experience
+
+#### Ed Anderson 
+Ed has three years of software engineering experience developing and renovating the full stack of an enterprise scale system. He holds a B.S. in Computer Science from the University of Pittsburgh, where his Capstone requirement included implementing a MVP token concept, PittCoin, which relied on a bounty-program to connect students to community-sourced homework solutions, the tokens earned from which could be exchanged for extra credit from their professors. He also has five years of experience as the creator and manager of the Katz Business Research Center, during which he oversaw the implementation and successful completion of over one thousand market research and consumer behavior studies and focus groups. 
+
+#### Matt Dennis
+Matt is an entrepreneur with over 10 years of sales and operational experience in the Computer Security and Blockchain industry. He holds a B.s. in Computer Forensics and Information Security, and as certifications in multiple levels of the web3 tech stack. 
+
+#### Will Chastka
+Will has a passion for gaming, ever since his first gaming console, a sega Saturn, playing virtual fighter 2 and Shinobi Legends, to Starcraft and DOTA. His passion for crypto extends decentralized finance, gaming, and many other applications. He previously held position as Community and Marketing Manager for CryptoAquatics NFT project, and holds a Master's in Business administration from the University of Pittsburgh and a M.S. in Business Analytics from Carnegie Mellon University. 
+
+We plan to leverage all of our experience and industry connections to build a heavily engaged community to gain market share through exploiting an emerging niche in the market with improved technology, our parachain. 
+
+### Team Code Repos
+
+- https://github.com/eca20/Coursework/tree/main/PittCoin
+- https://github.com/eca20/Coursework/tree/main/CS1699-Blockchain
+- https://github.com/eca20/BlockchainiaJRPG
+- https://github.com/eca20
+- https://github.com/bluedragonblockchain
+
+### Team LinkedIn Profiles (if available)
+
+- https://www.linkedin.com/in/edanderson4hire/
+- https://www.linkedin.com/in/wchastka/
+
+
+## Development Status
+We have started created a Unity game, but in C#. We are converting these scripts to Rust in the most performant way possible. 
```

Comment:
> We are converting these scripts to Rust in the most performant way possible.

Are you still going to use Unity here?

Reply:
Once the scripts work appropriately to control the game in C#, we will begin converting them to Rust. Our goal is to not to slow the creative flow required for the successful development of a first-person shooter multiplayer game while we experiment with our substrate node, so they are being developed separately. We will rely on unit testing throughout the development cycle to test the functionality of the server and node. The last few sprints in the roadmap will include converting the C# scripts produced by Unity into Rust scripts that interact with our blockchain.